src/pudl/metadata/resources/sec10k.py

```diff
@@ -187,22 +210,22 @@
         "field_namespace": "sec",
     },
     "core_sec10k__changelog_company_name": {
-        "description": (
-            """A historical record of the names each SEC 10-K filer has used.
-
+        "description": {
+            "additional_summary_text": "the names each SEC 10-K filer has used.",
+            "additional_details_text": """
 This table is extracted from the same SEC 10-K filing header information as
-``core_sec10k__quarterly_company_information``. Each filing reports the full history of
+:ref:``core_sec10k__quarterly_company_information``. Each filing reports the full history of
```

Comment:
right? i could be wrong....
```suggestion
:ref:`core_sec10k__quarterly_company_information`. Each filing reports the full history of
```